### PR TITLE
config: add release-7.4 branch config

### PR DESCRIPTION
**Pull request by purelind**
add release-7.4 branch config.

### CODE REVIEW

File: prow/config/config.yaml [hunk 1]
```diff
@@ -2295,6 +2390,7 @@ tide:
         - release-7.1 # LTS: avoid prow's "Merging to branch *** is forbidden" description issue.
         - release-7.2 # DMR: avoid prow's "Merging to branch *** is forbidden" description issue.
         - release-7.3 # DMR: avoid prow's "Merging to branch *** is forbidden" description issue.
```
Comment (wuhuizuo) [inline prow/config/config.yaml:2392]: We can delete the these released DMR branches.

Comment (wuhuizuo) [reply prow/config/config.yaml:2392]: We just need to keep the latest DMR branches

File: prow/config/external_plugins_config.yaml [hunk 2]
```diff
@@ -104,6 +104,7 @@ ti-community-label:
       - "needs-cherry-pick-release-7.1"
       - "needs-cherry-pick-release-7.2"
       - "needs-cherry-pick-release-7.3"
```
Comment (wuhuizuo) [inline prow/config/external_plugins_config.yaml:106]: Please delete these labels, same as branches, we just need keep the latest DMR's  need-cherry-pick label.

Comment (wuhuizuo) [reply prow/config/external_plugins_config.yaml:106]: Or make it updated in another PR.

File: prow/config/config.yaml (hunk 1 above)
Comment (purelind) [reply prow/config/config.yaml:2392]: Will create a separate PR to handle the deletion of DMR-related content.

File: prow/config/external_plugins_config.yaml (hunk 2 above)
Comment (purelind) [reply prow/config/external_plugins_config.yaml:106]: Will create a separate PR to handle the deletion of DMR-related content.

**Review by wuhuizuo — APPROVED** [review]
/lgtm
/approve

**Pull request merged**